examples/collection-get-item/index.tsx

```diff
@@ -0,0 +1,26 @@
+import "./style.css";
+import * as Ariakit from "@ariakit/react";
+
+type CustomItem = {
+  id: string;
+  element?: HTMLElement | null | undefined;
+  color?: string;
+};
```

Comment:
I think It would be better if I can just import `CollectionStoreItem` to extend it here, but it seems `@ariakit/react` doest not export it.

Reply:
Yes, we should definitely consider exporting that. I just want to see more use cases emerge to ensure it's the right API for TypeScript.